code/modules/projectiles/guns/revolvers.dm

```diff
@@ -353,7 +353,7 @@
 		/obj/item/attachable/lace,
 	)
 	attachable_offset = list("muzzle_x" = 33, "muzzle_y" = 19,"rail_x" = 13, "rail_y" = 23, "under_x" = 22, "under_y" = 14, "stock_x" = 22, "stock_y" = 19)
-	fire_delay = 0.15 SECONDS
```

Comment:
why is this changed with 0 mention now

Reply:
I'll remove it and it can be worked on in another PR